testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/archives/TestNonReproducibleArchives.groovy

```diff
@@ -25,10 +25,10 @@ import java.lang.annotation.RetentionPolicy
 import java.lang.annotation.Target
 
 /**
- * A Spock extension to run tests with reproducible archives enabled.
+ * A Spock extension to run tests with non-reproducible archives.
```

Comment:
```suggestion
 * A Spock extension to run tests the second time using non-reproducible archives.
```

We explain it further in the javadoc, but being explicit here does not hurt. I think, it's important to have this in the first line of the javadoc, because the annotation name reads as if we _only_ running the tests with one setup or the other (imagine, looking at the usage of this annotation by the eyes of an external contributor). If we keep the annotation naming pattern the same, the javadoc is the next best place to clarify the effect.

Also, to use this as an example of the alternative language I suggested previously, consider how this would read:
```suggestion
 * A Spock extension to run tests the second time using file system-sensitive archives.
```